app/javascript/chat/__tests__/message.test.jsx

```diff
@@ -38,4 +39,20 @@ describe('<Message />', () => {
 
     expect(profileLink.parentElement).toHaveStyle({ color: msg.color });
   });
+
+  it('should highlight @mentions to the logged in user', () => {
+    const testMessage = {
+      username: 'testUser',
+      user_id: 456,
+      message: "<p>hello <a href='testUser'>@testUser</a></p>",
+    };
+
+    const { getByText } = render(
+      getMessage(testMessage, { currentUserId: testMessage.user_id }),
+    );
+
+    const profileLink = getByText(`@${testMessage.username}`);
+
+    expect(profileLink.parentElement.innerHTML).toMatch(/<mark>.*<\/mark>/i);
```

Comment:
blocking: We want to ensure the username is contained within the `<mark />`. Otherwise `<mark></mark>` would pass or e.g. `<mark>hello</mark>`.
 
```suggestion
    expect(profileLink.parentElement.innerHTML).toMatch(new RegExp(`<mark>@${testMessage.username}</mark>`, 'i'));
```

Reply:
Great catch. Fixed :slightly_smiling_face: